src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -145,6 +145,9 @@
     <StandardSourceBuildArgs>$(StandardSourceBuildArgs) /p:PreviouslySourceBuiltNupkgCacheDir="$(PrebuiltSourceBuiltPackagesPath)"</StandardSourceBuildArgs>
     <StandardSourceBuildArgs>$(StandardSourceBuildArgs) /p:SourceBuildUseMonoRuntime=$(SourceBuildUseMonoRuntime)</StandardSourceBuildArgs>
 
+    <!-- https://github.com/dotnet/source-build/issues/3081 -->
+    <StandardSourceBuildArgs>$(StandardSourceBuildArgs) /p:CheckEolTargetFramework=false</StandardSourceBuildArgs>
```

Comment:
This won't work in most if not all cases.  The reason is that build properties don't get automatically flown into the repo inner builds.  An environment variable ([example](https://github.com/dotnet/installer/pull/15222/files#diff-eeec14bcfae04e9276d77a77ecc85725588e875782ac908963dde2e3cf4d340aR115)) would be the way to accomplish this given the way the system currently works.

Reply:
This [documentation](https://github.com/dotnet/arcade/tree/main/src/Microsoft.DotNet.Arcade.Sdk/tools/SourceBuild#msbuild-execution) describes how the source-build works for a repo.  The outer build properties do not flow into the inner build.

Reply:
@MichaelSimons - Does this mean that the original [content that I removed from xliff-tasks](https://github.com/dotnet/installer/pull/15222/files#diff-f913879d973c98706d4621c6e736a58d75e055cfe8c2be18e3c7a4f0224d8f5e) wasn't actually working as intended?

Reply:
It means that in main/.NET 8.0, they removed the EOL TFMs that were causing the build to break.  If you were to backport these changes to .net 7.0 or 6.0, I would suspect you would get a build failure.

Reply:
...continuing that thought.  I am sure you would get a build failure in FSharp.  I can't answer why the xliff-tasks pattern works without looking at a binlog.